handler.py

```diff
@@ -14,38 +16,120 @@
 SQS_CLIENT = boto3.client('sqs')
 
 
-def runScheduledObservatoryScan(event, context):
+def addPortScanToQueue(event, context):
+    data = json.loads(event['body'])
+    target = Target(data.get('target'))
```

Comment:
If you are worried about having the '.isValid()' step for each scanner implementation, we could just  make validity checking part of the Objects instantiation.  The added work would then just move to our unit-tests, which would have to catch exceptions for invalid target examples.  Just a thought.

Reply:
I don't know what you mean here, do you mean as a part of __init__ for Target class?

Reply:
Nevermind, I figured it out. Will update the Target object to make the validity checking upon instantiation.